test/kickstart-templates/includes/post-system.cfg

```diff
@@ -1,9 +1,16 @@
 # Create a default redhat user, allowing it to run sudo commands without password
-useradd -m -d /home/redhat -p \$5\$XDVQ6DxT8S5YWLV7\$8f2om5JfjK56v9ofUkUAwZXTxJl3Sqnc9yPnza4xoJ0 redhat
+useradd -m -p \$5\$XDVQ6DxT8S5YWLV7\$8f2om5JfjK56v9ofUkUAwZXTxJl3Sqnc9yPnza4xoJ0 redhat
 echo -e 'redhat\tALL=(ALL)\tNOPASSWD: ALL' > /etc/sudoers.d/microshift
 
 # Make the KUBECONFIG from MicroShift directly available for the root user
-echo -e 'export KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig' >> /root/.profile
+if [ ! -d /root ]; then
+    # Workaround for bootc container init. /var/roothome (which /root symlinks to) is not yet present.
```

Comment:
I think systemd or some service sets it all up. If we create it here, it'll stay.
That file doesn't exist on normal install anyway, so it's not like we lose any content that might be there.

Also, was the intention that a manual `source ~/.profile` is needed each time? Because it's not automatically sourced when logging using `sudo su`

Reply:
Does it not work with `sudo -i`? 
P.S. `sudo su -` should be the `su` equivalent.

Reply:
In any case, let's expand on the comment, saying that we expect the file to be there after the /root is mounted.